src/app/clusters/color-control-server/color-control-server.cpp

```diff
@@ -431,15 +431,16 @@ ColorControlServer & ColorControlServer::Instance()
     return instance;
 }
 
+#if defined(MATTER_DM_PLUGIN_SCENES_MANAGEMENT)
```

Comment:
nit: just for consistency. you use #ifdef most of the time but for chip::scenes::SceneHandler * declarations you use #if defined()